plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -166,13 +166,14 @@ export class PersonState {
         const props = { ...propertiesOnce, ...properties }
         const propertiesLastOperation: Record<string, any> = {}
         const propertiesLastUpdatedAt: Record<string, any> = {}
+        const timestampIso = createdAt.toISO()
```

Comment:
Just below we're using `this.timestamp` not createdAt. Why use one over the other? Can we make this change in a way that doesn't increase confusion around timestamping?

Reply:
well the way would be to remove `createdAt` from this function